.pre-commit-config.yaml

```diff
@@ -8,12 +8,11 @@ repos:
       - id: end-of-file-fixer
       - id: check-yaml
       - id: check-added-large-files
-  - repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v2.0.2
+  - repo: https://github.com/hhatto/autopep8
+    rev: v2.0.4
     hooks:
       - id: autopep8
-        args: [--max-line-length=120, -a, -i, -r, .]
   - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+    rev: 5.13.2
```

Comment:
isort & all or should we switch to all-in-ruff asap ? :thinking:

Reply:
I guess upgrading the surrounding tooling could warrant its own PR :)